packages/util/src/dashify.ts

```diff
@@ -0,0 +1,10 @@
+export function dashify(str?: string) {
+  if (!str) return
+
+  return str
+    .trim()
+    .replace(/([a-z])([A-Z])/g, '$1-$2')
+    .replace(/([a-z])([0-9]+)/g, '$1-$2')
+    .replace(/([0-9])([a-z])/g, '$1-$2')
+    .toLowerCase()
+}
```

Comment:
I think we should be doing this manually.

Reply:
I don't know exactly what you mean by your comment. I think I need more context.